src/api/controllers/authController.js

```diff
@@ -34,6 +35,14 @@ export default class Auth {
       email,
       password: hashedPassword
     });
+    await mailer('Please verify your email', 'Email verification', email, 'notifications', {
+      email,
+      buttonText: 'Verify',
+      userName: username,
+      message: 'Please click on the link to verify your email for authors haven,If you didn\'t request this, simply ignore this e-mail.',
```

Comment:
Nitpick here. @truestbyheart ensure your texts are properly punctuated. Like so:

`"Please click on the link to verify your email for Authors Haven. If you didn't request this, simply ignore this email."`